catalogue/webapp/components/WorkDetails/WorkDetails.tsx

```diff
@@ -593,7 +599,9 @@ const WorkDetails: FunctionComponent<Props> = ({ work }: Props) => {
         </WorkDetailsSection>
       )}
 
-      {digitalLocation && (locationOfWork || encoreLink) && <WhereToFindIt />}
+      {digitalLocation && (locationOfWork || showEncoreLink) && (
```

Comment:
Maybe for v2 of the PR, could we have a state handler that tells us what these all mean similar to the [`getItemLinkState`](https://github.com/wellcomecollection/wellcomecollection.org/blob/5f0c5f0bf3b7370ee943131e27cdb1dc130d9ad1/catalogue/webapp/components/WorkDetails/WorkDetails.tsx#L52-L72)?